wagtail/blocks/struct_block.py

```diff
@@ -17,24 +17,36 @@
 
 
 class StructBlockValidationError(ValidationError):
```

Comment:
Shall we add this to `__all__`? We do it for `StreamBlockValidationError` so it can be imported from `wagtail.blocks`